### PR TITLE
[generate][m]: Generated descriptor.json from inline csv - refs #10

### PR DESCRIPTION
**Pull request by Mikanebu**
* Created function for generating descriptor from tabular data.
* Included unittest.

### CODE REVIEW

**Review by anuveyatsu — APPROVED** [review]
OK

File: test/test_descriptor.js [hunk 1]
```diff
@@ -0,0 +1,37 @@
+const assert = require('assert')
+const generateDescriptor = require('../lib/descriptor.js').generateDescriptor
+
+describe('generate dp', () => {
+  it('from inline data', function() {
+    const csvData = "a,b,c\n1,2,3\n4,5,6"
```
Comment (zelima) [inline test/test_descriptor.js]: can we have something else besides integers Eg: date, string.. so we could be sure that types for fields are set correctly

File: lib/descriptor.js [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+    name: dpName,
+    resources: [
+      {
+        format: "csv",
+        data: data
+      }
+    ]
+  }
+  
+  await parse(data, (error, values) => {
```
Comment (zelima) [inline lib/descriptor.js:15]: can we have the error handler as well, Eg: 
```
if (error) {
  process.stdout.write(error.message) // or whatever reasonable error handling would be
}
```

File: test/test_descriptor.js [hunk 3]
```diff
@@ -0,0 +1,37 @@
...
+          title: '',
+          description: '',
+          type: 'integer',
+          format: 'default'
+        }
+      ]
+     }
+    const out = generateDescriptor(csvData).then(val => {
+      assert.equal(val.name, 'scratchpad')
+      assert.equal(JSON.stringify(val.resources[0]['schema']), JSON.stringify(expectedSchema))
```
Comment (zelima) [inline test/test_descriptor.js]: think simple assert.deepEqual() would be enough here

```
assert.deepEqual(val.resources[0]['schema'],expectedSchema)
```

File: test/test_descriptor.js (hunk 1 above)
Comment (Mikanebu) [reply test/test_descriptor.js]: Good point, I will add them

File: test/test_descriptor.js (hunk 3 above)
Comment (Mikanebu) [reply test/test_descriptor.js]: ok, thanks

**Comment by zelima** [comment]
Looks better

**Pull request merged**